XamlStyler.Package/Helpers/OptionsHelper.cs

```diff
@@ -0,0 +1,111 @@
+﻿// © Xavalon. All rights reserved.
+
+using EnvDTE;
+using Microsoft.VisualStudio.Shell;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Xavalon.XamlStyler.Core.Options;
+using Xavalon.XamlStyler.Package.Extensions;
+
+namespace Xavalon.XamlStyler.Package.Helpers
+{
+    public sealed class OptionsHelper
+    {
+        private readonly StylerPackage package;
+
+        public OptionsHelper(StylerPackage package)
+        {
+            this.package = package;
+        }
+
+        public IStylerOptions GetGlobalStylerOptions()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+            return this.package.GetDialogPage(typeof(PackageOptions)).AutomationObject as IStylerOptions;
+        }
+
+        public IStylerOptions GetDocumentStylerOptions(Document document)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+            Properties xamlEditorProps = this.package.IDE.Properties["TextEditor", "XAML"];
```

Comment:
Shouldn't it use `Constants.XamlLanguageType` or these things are not the same?

Reply:
From what I could gather, they are not quite the same -- one is the language type of Document class (the constant), while this is an editor window of type XAML. It might be a distinction without a difference, though.